### DISCUSSION TITLE
Preserve hotkey to toggle HDR - Windows 11

### DISCUSSION THREAD

**Discussion by Shiverous** [midrare/hyperenable#1] [Ideas]
Hello! I really like the concept of Hyperenable, but was wondering if it were possible to have the ability to toggle which shortcuts still remain active and which ones don't. Perhaps a GUI or some kind of settings.txt file?

See - when I watch movies or other content with really dark scenes, I need to turn on Windows HDR so I can see the content on my dimly lit monitor. I use the `Win + Alt + B` shortcut, which seems to be disabled with Hyperenable?

The ability to disable crappy shortcuts like `Win + Shift + Ctrl + Alt + L` for opening LinkedIn, is invaluable as a power-user, but a bit of customization would go a long way in my opinion.

Cheers!

**Comment by midrare** [comment]
Yes, this was something I originally wanted to do for v0.1.0 but didn't. It's still on the TODO list though. (P.S. I've converted this discussion to an issue: #2)
